src/lib.rs

```diff
@@ -53,7 +53,7 @@
 //! notation.
 
 #![no_std]
-#![doc(html_root_url = "https://docs.rs/ryu-js/0.2.2")]
+#![doc(html_root_url = "https://docs.rs/ryu-js/1.0.0")]
```

Comment:
Maybe we should just point to the latest version here, without pinning

Reply:
I think it's better to remove it altogether. See https://github.com/rust-lang/api-guidelines/pull/230